bangazonapi/models/product.py

```diff
@@ -61,9 +61,12 @@ def average_rating(self):
         total_rating = 0
         for rating in ratings:
             total_rating += rating.rating
+        try:
+            avg = total_rating / len(ratings)
+            return avg
+        except:
```

Comment:
Since you have this exception here you don't need the other ones in the other files. Like the other ones though this one should have the `ZeroDivisionError as ex`